modules/coreboot

```diff
@@ -8,6 +8,7 @@ else ifeq "$(CONFIG_COREBOOT_VERSION)" "4.12"
 	coreboot_version := 4.12
 	coreboot_hash := edcad000ee9b73183c396ea76155629b3d27c693e0f1ae83e3424c4d936e2be2
 	coreboot-blobs_hash := 4735ee6850d55d1e65dee8b08cc9b28b8af00b42acf31365f5d9545406579104
+	coreboot_depends := $(if $(CONFIG_PURISM_BLOBS), purism-blobs)
```

Comment:
@MrChromebox We can chainload multiple if here for different board configurations?

```
coreboot_depends := $(if $(CONFIG_PURISM_BLOBS), purism-blobs)
coreboot_depends := $(if $(CONFIG_xx30_BLOBS),xx30-blobs)
```

etc?

Reply:
> @MrChromebox We can chainload multiple if here for different board configurations?

would need to be a single line I believe:
`coreboot_depends := $(if $(CONFIG_PURISM_BLOBS), purism-blobs) $(if $(CONFIG_xx30_BLOBS),xx30-blobs)`